src/components/v2/V2Project/V2FundingCycleSection/index.tsx

```diff
@@ -31,6 +36,13 @@ export default function V2FundingCycleSection({
   } = useContext(ThemeContext)
   const { fundingCycle, isPreviewMode } = useContext(V2ProjectContext)
 
+  const canReconfigure = useHasPermission(V2OperatorPermission.RECONFIGURE)
+  const showReconfigureButton = canReconfigure && !isPreviewMode
+
+  if (!fundingCycle) {
```

Comment:
**issue**: can we add a loading flag for `fundingCycleLoading` in V2PorjectContext?

Reply:
Yep